### ISSUE TITLE
Is it possible to send click events to children views of a header?

### ISSUE THREAD

**Issue by anothem** [timehop/sticky-headers-recyclerview#81]
Let's say in my header layout I place a button. Is there a way to allow that button to capture clicks/touch events?

**Comment by jacobtabak** [comment]
Not directly, since the headers are really just decorations and not actual views. One of the limitations of the library :(

**Comment by kesari** [comment]
hi,  I understand that since these are decorations you cant click on individual items, but are there any other indirect methods to get these clicks ?

**Comment by jacobtabak** [comment]
Yes, but not easily. You could use [view.dispatchTouchEvent()](https://developer.android.com/reference/android/view/View.html#dispatchTouchEvent(android.view.MotionEvent)).

**Comment by ivamluz** [comment]
Hi guys,

Were you able to implement this? We have a scenario where a "MORE" button is placed inside the header and, when clicked, a new activity is displayed, showing all items.

I took a look at view.dispatchTouchEvent(), as pointed by @jacobtabak, but couldn't find out how to use it (I'm still a newbie in the platform :( )

**Comment by fffffz** [comment]
I have resolved this issue :)  see pull requests

**Comment by wangkunhui** [comment]
@iceGeneral Can it visual feedback when I touch the button in header?

**Comment by fffffz** [comment]
@wangkunhui Sorry, I also don't know how to solve it
